### PR TITLE
Project details

### PR DESCRIPTION
**Pull request by Bavon101**
- [ ] Dynamically add Project object as HTML elements
- [ ] Create Projects object list
- [ ] Create project pop up view
- [ ] Show project pop up on view click

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -92,8 +76,10 @@ <h1 class="name">Akumu Bavon</h1>
       <div class="image_dots"></div>
       <div class="bottom_geometry"></div>
     </section>
+    <div class="project_details" id="view_project">
 
-    <section class="project_section" id="Portfolio">
+    </div>
+    <section class="project_section can_dissapear" id="Portfolio">
```
Comment (Meltrust) [inline index.html:82]: - [x] Kindly please implement the modals on the projects section on the desktop view, as stated on the requirements:

image

This is the desktop popup implementation on the Figma:

image


That way, your portfolio will have popups both on desktop and mobile.  You could reuse the logic that you already implemented for the mobile! 💪

File: index.html [hunk 2]
```diff
@@ -457,12 +271,13 @@ <h2 class="about_title">About</h2>
         <div class="dot_image_with_teal"></div>
       </div>
     </section>
-    <section class="contact_section" id="Contact">
+    <section class="contact_section can_dissapear" id="Contact">
```
Comment (Meltrust) [inline index.html:164]: - [x] This bug was introduced on your desktop view at some point, please kindly fix it:

image

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Bavon101, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Beautiful project! Amazing job! Just make the small changes requested and it should be approved really fast 💪 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Bavon101) [reply index.html:164]: @Meltrust Thank you for catching this, I'm not able to reproduce this issue, the difference might be based on the browser. I would appreciate it if you could specify the browsers you were using 🐱‍👤.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Bavon101, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Meltrust) [reply index.html:164]: You are very right!  In Chrome it works, but not on Firefox.  I had to remove the width on the element in order for it to work:

image

Thanks for pointing this out.

Comment (Bavon101) [reply index.html:164]: @Meltrust Thank you once again for catching this, I'll implement the change 🐱‍🏍.

**Pull request merged**